.github/workflows/release-please.yml

```diff
@@ -18,3 +18,20 @@ jobs:
           release-type: node
           package-name: unchained-wallets
           command: manifest
+
+      # The logic below handles the npm publication:
+      - uses: actions/checkout@v2
+        # these if statements ensure that a publication only occurs when
+        # a new release is created:
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
I can change it, what should I set it to?

Reply:
We put our engines at:

```
  "engines": {
    "node": ">=16"
  },
```
I guess that's not terribly relevant here though since it's not really related, and 16 is about to hit EOL. So, 18? 

I'm also having trouble understanding what this field even is for, and also if we should use a later version of the action: https://github.com/actions/setup-node. 🤔 What does your gut say? I kind of want to sya bump up to setup-node@v3 and then try with node-version 18 but I'm not sure.

Reply:
Yeah, let's do node-version 18 and setup-node@v3

Reply:
Okay, done!